webcompat/static/js/lib/bugform.js

```diff
@@ -573,6 +605,136 @@ function BugForm() {
     });
   };
 
+  // functions for Image Editor
+  this.openImageEditor = function() {
+    //var doc = global.document;
+    var canvas = document.createElement('canvas');
+    var ctx = canvas.getContext('2d');
+    var img = new Image();
+    canvas.id = "mCanvas";
+    canvas.width = 500;
+    canvas.height = 500;
+    img.src = this.dataURI;
+
+    var modalContent = document.querySelector('.modal-content');
+
+    this.editImageModal.removeClass("is-hidden");
+    modalContent.appendChild(canvas);
+
+    // TODO ask Mike if there is a better way to share this globally, like this.ctx
+    globalCtx = ctx;
```

Comment:
@miketaylr I kind of understand global vars are not good but this ctx needs to be shared. should I be using something like this.ctx instead?